DQM/L1TMonitorClient/python/L1TStage2MonitorClient_cff.py

```diff
@@ -45,6 +45,7 @@
                       + l1tStage2uGMTClient
                       + l1tStage2uGTClient
                       + l1tStage2EMTFEventInfoClient
+                      + l1tObjectsTimingClient
```

Comment:
You need to add back the loading of the cff before using `l1tObjectsTimingClient`. This was reoved accidentally with this commit https://github.com/cms-sw/cmssw/pull/22524/commits/fea3404d6194626f3368d6f606172dfc67fd3a2b#diff-dde332057b113753b39779429d0f48b0L19
```
# L1T Objects Ration Timing Plots
from DQM.L1TMonitorClient.L1TObjectsTimingClient_cff import *
```